tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -287,7 +287,7 @@ def test_5_full_snapshots(network_config,
     # - Rootfs: Ubuntu 18.04
     # - Microvm: 2vCPU with 512 MB RAM
     # TODO: Multiple microvm sizes must be tested in the async pipeline.
-    microvm_artifacts = ArtifactSet(artifacts.microvms(keyword="2vcpu_512mb"))
+    microvm_artifacts = ArtifactSet(artifacts.microvms(keyword="2vcpu_256mb"))
```

Comment:
Just curious, what is the reason for this?

Reply:
Reduce full snapshot create duration. It depends on the memory size as we dump it in full. For diff snapshots we just dump the dirty pages no matter the size.

Reply:
Reduce full snapshot create duration. It depends on the memory size because we dump all pages. Diff create snapshot duration is not influenced by memory size as we only dump the dirty pages.